packages/article-converter/src/plugins/embed/campaignBlockPlugin.tsx

```diff
@@ -24,7 +24,7 @@ export const campaignBlockPlugin: PluginType = (element, _, opts) => {
       imageSide={embed.imageSide}
       image={
         data.status === 'success' && data.data.image
-          ? { src: data.data.image.image.imageUrl, alt: data.data.image.alttext.alttext }
+          ? { src: data.data.image.image.imageUrl, alt: embed.alt ? embed.alt : data.data.image.alttext.alttext }
```

Comment:
Vil forresten dette virke med alt="" altså tom alt-tekst? For det var jo ønske om at for kampanjeblokk skulle det være mogeleg å sette bildet som dekorativt, og tom alt skulle dekke det.